pkg/harvester/config/config.go

```diff
@@ -39,7 +39,7 @@ func new(configBytes []byte) (*HarvesterConfig, error) {
 	var config HarvesterConfig
 
 	if err := hcl.Decode(&config, string(configBytes)); err != nil {
-		return nil, fmt.Errorf("unable to decode configuration: %v", err)
+		return nil, errors.New(fmt.Sprintf("unable to decode configuration: %v", err))
```

Comment:
nit: fmt.Errorf is a shortcut for errors.Next + fmt.Sprintf
```suggestion
		return nil, fmt.Errorf("unable to decode configuration: %v", err)
```

Reply:
🤦‍♀️